avo-hq/avo
# feature: durable and bookmarkable tabs

# Description


Implements https://github.com/avo-hq/avo-3/issues/432
Superseeds https://github.com/avo-hq/avo-3/pull/433

I chose to remember the selected tab in localstorage as it's more durable in the long run and improves the experience throughout.

In this PR I removed an extra `turbo_frame` that was wrapping each tab. This reduces the complexity more.
The association fields will still keep their lazy-loading behavior.



# Checklist:



## Screenshots


## Manual review steps


1. Go to a user
1. Select one tab
2. notice the param change (bookmarkable)
3. navigate somewhere else
4. navigate back using the absolute URL (without the tab group param)
5. notice the last tab was selected (durable)

Manual reviewer: please leave a comment with output from the test if that's the case.
